From 15480d7250bf39b019f6473b9338fcff3a5f129a Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Thu, 19 Jun 2014 21:10:58 +0200
Subject: [PATCH] [DEV] open new 1.6 development branch

This new branch is based on 1.5.0, which 1.6-dev0 is 100% equivalent to.
The README has been updated to mention that it is a development branch.

Released version 1.6-dev0 with the following main changes :
    - exact copy of 1.5.0
---
 CHANGELOG             |  3 +++
 README                | 16 +++++++++-------
 VERSION               |  2 +-
 doc/configuration.txt |  2 +-
 examples/haproxy.spec |  5 ++++-
 5 files changed, 18 insertions(+), 10 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index a893627..f17e680 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,6 +1,9 @@
 ChangeLog :
 ===========
 
+2014/06/19 : 1.6-dev0
+    - exact copy of 1.5.0
+
 2014/06/19 : 1.5.0
     - MEDIUM: ssl: ignored file names ending as '.issuer' or '.ocsp'.
     - MEDIUM: ssl: basic OCSP stapling support.
diff --git a/README b/README
index 424a1f8..bf9e82b 100644
--- a/README
+++ b/README
@@ -1,7 +1,7 @@
                          ----------------------
                              HAProxy how-to
                          ----------------------
-                               version 1.5
+                             version 1.6-dev
                              willy tarreau
                                2014/06/19
 
@@ -9,12 +9,14 @@
 1) How to build it
 ------------------
 
-First, please note that this version is a stable version, so in general if you
-are not used to build from sources, it is recommended that instead you follow
-the packaged updates provided by your software vendor or Linux distribution.
-Most of them are taking this task seriously and are doing a good job. If for
-any reason you'd prefer a different version than the one packaged for your
-system, or to get some commercial support, other choices are available at :
+First, please note that this version is a development version, so in general if
+you are not used to build from sources or if you don't have the time to track
+very frequent updates, it is recommended that instead you switch to the stable
+version (1.5) or follow the packaged updates provided by your software vendor
+or Linux distribution. Most of them are taking this task seriously and are
+doing a good job. If for any reason you'd prefer a different version than the
+one packaged for your system, or to get some commercial support, other choices
+are available at :
 
                         http://www.haproxy.com/
 
diff --git a/VERSION b/VERSION
index bc80560..0b9d9ed 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-1.5.0
+1.6-dev0
diff --git a/doc/configuration.txt b/doc/configuration.txt
index 5ee2ae8..5f80674 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -2,7 +2,7 @@
                                 HAProxy
                           Configuration Manual
                          ----------------------
-                              version 1.5
+                              version 1.6
                              willy tarreau
                               2014/06/19
 
diff --git a/examples/haproxy.spec b/examples/haproxy.spec
index 677ee0a..a22e7ea 100644
--- a/examples/haproxy.spec
+++ b/examples/haproxy.spec
@@ -1,6 +1,6 @@
 Summary: HA-Proxy is a TCP/HTTP reverse proxy for high availability environments
 Name: haproxy
-Version: 1.5.0
+Version: 1.6-dev0
 Release: 1
 License: GPL
 Group: System Environment/Daemons
@@ -77,6 +77,9 @@ fi
 
 %changelog
 * Thu Jun 19 2014 Willy Tarreau <w@1wt.eu>
+- updated to 1.6-dev0
+
+* Thu Jun 19 2014 Willy Tarreau <w@1wt.eu>
 - updated to 1.5.0
 
 * Wed May 28 2014 Willy Tarreau <w@1wt.eu>
-- 
1.8.5.5

